fix: return local time from `getFormattedDate` and `getCurrentDateTime`

crizCraig opened the pull request:
Currently `getFormattedDate` returns UTC and `getCurrentDateTime` returns a mix of local and UTC. This currently only affects prompt templates, but I have another change to send this to openai backends to better serve users by knowing their time.

crizCraig committed:
Return local time from `getFormattedDate` and `getCurrentDateTime` instead of UTC

tjbck commented:
`getCurrentDateTime` definitely returns local current date time, I'd suggest you try the latest version.

crizCraig commented:
Thanks Tim. Maybe I can use prompt variables for what I need. `getCurrentDateTime` in the latest dev actually _mixes_ local with UTC since `toISOString` returns UTC.

```
// Get the date in the format YYYY-MM-DD
export const getFormattedDate = () => {
	const date = new Date();
	return date.toISOString().split('T')[0];
};

// Get the time in the format HH:MM:SS
export const getFormattedTime = () => {
	const date = new Date();
	return date.toTimeString().split(' ')[0];
};

// Get the current date and time in the format YYYY-MM-DD HH:MM:SS
export const getCurrentDateTime = () => {
	return `${getFormattedDate()} ${getFormattedTime()}`;
};
```
